source/local/local.go

```diff
@@ -216,31 +245,48 @@ func parseFeatures(lines [][]byte, fileName string) map[string]string {
 			lineSplit := strings.SplitN(line, "=", 2)
 
 			key := lineSplit[0]
-
-			// Check if it's a boolean value
-			if len(lineSplit) == 1 {
-				features[key] = "true"
-			} else {
-				features[key] = lineSplit[1]
-			}
+			updateFeatures(features, &parsingOpts.IsIncludedInFeatures, key, lineSplit)
+			updateFeatures(labels, &parsingOpts.IsIncludedInLabels, key, lineSplit)
 		}
 	}
 
-	return features
+	return features, labels
+}
+
+func updateFeatures(m map[string]string, shouldBeIncluded *bool, key string, lineSplit []string) {
```

Comment:
With option names changed to `skipLabel` / `skipFeature`, argument name should also be changed:
```suggestion
func updateFeatures(m map[string]string, skip *bool, key string, lineSplit []string) {
```

(And function logic regarding the bool inverted accordingly.)

Reply:
Please set this as resolved (as I do not have option for that).